srcpkgs/grass/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'grass'
 pkgname=grass
```

Comment:
https://grasswiki.osgeo.org/wiki/Compile_and_Install seems to indicate it's possible to use python 3. can we try moving to it?

Reply:
I already have a draft template for this package, yet the outcome proved to be challenging.

I will try to update grass and include it in this pull request instead of the bumped one.

Reply:
@classabbyamp, I could update it to 8.2.0 (did not crosscompile yet), but apparently it hangs with wxPython4 (`ToolBar.AddTool(): arguments did not match any overloaded call: ...`), rendering grass inoperable.

According to https://github.com/OSGeo/grass/issues/2019, this can be solved by updating wxPython4 to the newer version, but here the old version of wxWidgets itself (3.0.5 vs. 3.2.1) makes its appearance:

```
In file included from ../../../../sip/cpp/sip_corewxZoomGestureEvent.cpp:10:
../../../../sip/cpp/sipAPI_core.h:22411:10: fatal error: wx/collheaderctrl.h: No such file or directory
22411 | #include <wx/collheaderctrl.h>
      |          ^~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

Given how many packages depend on wxWidgets, I suspect that it must be updated in a separate PR along with its reverse dependencies.

Is there a course of action you would recommend?

Reply:
i think it's fine to leave updating grass to py3 alone for now, just the bump is fine. it's evident that it's a much bigger project than just switching out deps. just that you're working on it is enough for me :)

Reply:
just for fun, I had a go at updating grass:

```patch
diff --git a/srcpkgs/grass/template b/srcpkgs/grass/template
index bccd691a81..98b733528b 100644
--- a/srcpkgs/grass/template
+++ b/srcpkgs/grass/template
@@ -1,30 +1,30 @@
 # Template file for 'grass'
 pkgname=grass
-version=7.6.1
-revision=4
+version=8.2.0
+revision=1
 _binver=${version//./}
 _binver=${_binver:0:2}
 build_style=gnu-configure
-configure_args="--prefix=\${DESTDIR}/usr/share --bindir=\${DESTDIR}/usr/bin
+configure_args="--prefix=/usr/share --bindir=/usr/bin
  --with-freetype-includes=${XBPS_CROSS_BASE}/usr/include/freetype2"
-hostmakedepends="flex libgdal-tools pkg-config python-numpy tar"
+hostmakedepends="flex libgdal-tools pkg-config python3-numpy tar"
 makedepends="proj-devel tiff-devel libgdal-devel sqlite-devel
- fftw-devel cairo-devel glu-devel wxPython-devel"
-depends="python-numpy wxPython"
+ fftw-devel cairo-devel glu-devel wxPython4 python3-six"
+depends="python3-numpy wxPython4 python3-six"
 short_desc="Geographic Resources Analysis Support System - GIS"
 maintainer="Alex Jarosch <research@alexj.at>"
 license="GPL-2.0-or-later"
 homepage="https://grass.osgeo.org/"
 distfiles="https://grass.osgeo.org/grass${_binver}/source/${pkgname}-${version}.tar.gz"
-checksum=9e25c99cafd16ed8f5e2dca75b5a10dc2af0568dbedf3fc39f1c5a0a9c840b0b
-python_version=2 #unverified
+checksum=621c3304a563be19c0220ae28f931a5e9ba74a53218c5556cd3f7fbfcca33a80
+python_version=3
 nocross="tries to execute target binaries"
 
 post_install() {
-	# move grass76 libraries from /usr/share to /usr/lib
+	# move grass libraries from /usr/share to /usr/lib
 	vmkdir usr/lib
-	mv ${DESTDIR}/usr/share/grass* ${DESTDIR}/usr/lib
+	mv ${DESTDIR}/usr/share/grass${_binver} ${DESTDIR}/usr/lib
 	# fixes a todo in GISBASE
-	sed -i ${DESTDIR}/usr/bin/grass${_binver} \
-		-e "105s;\(gisbase =\).*;\1 \"/usr/lib/grass-${version}\";"
+	sed -i ${DESTDIR}/usr/bin/grass \
+		-e "91s;\(GISBASE =\).*;\1 \'/usr/lib/grass${_binver}\';"
 }
```
seems to compile fine (there were some errors in the log but idk it still builds), **however** it does not run because we need at least wxPython4.1, it seems 🙃 (https://github.com/OSGeo/grass/issues/2019)